uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -50,16 +50,17 @@ public class MsalTestApp extends App {
     public MsalTestApp() {
         super(MSAL_TEST_APP_PACKAGE_NAME, MSAL_TEST_APP_NAME, new LocalApkInstaller());
         localApkFileName = MSAL_TEST_APP_APK;
+        localUpdateApkFileName = MSAL_TEST_APP_APK;
```

Comment:
But why we need to have update apk file name assigned here in the default constructor？